go.mod

```diff
@@ -12,12 +12,15 @@ require (
 	github.com/coreos/pkg v0.0.0-20180928190104-399ea9e2e55f
 	github.com/docker/go-units v0.4.0
 	github.com/go-echarts/go-echarts v1.0.0
+	github.com/go-openapi/spec v0.20.3 // indirect
```

Comment:
Can you try `go get -d github.com/pingcap/tidb-dashboard@v2021.05.12.1 && make pd-server && go mod tidy` ? I see some `indirect`.

Reply:
`go get -d` was awesome!